src/main/java/core/basesyntax/WorkWithFile/FileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.WorkWithFile;
+
+import core.basesyntax.WorkWithFileInterface.FileDataReader;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileDataReader {
+    @Override
+    public List<String> read(String fileName) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(fileName))) {
+            reader.readLine(); // Пропускаем заголовок
+            String line;
+            while ((line = reader.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Consider replacing `e.printStackTrace();` with a more informative logging mechanism or rethrowing the exception with a custom message. This will help in identifying the issue more effectively in a production environment.